pages/common/silicon.md

```diff
@@ -0,0 +1,16 @@
+# silicon
+
+> Create beautiful image of your source code.
+> More information: <https://github.com/Aloxaf/silicon/>.
+
+- Generate image from source file:
+
+`silicon  {{path/to/code.py}} -o {{path/to/image.png}}`
+
+- Generate image from source file with special program language syntax highlighting:
+
+`silicon  {{path/to/code.py}} -o {{path/to/image.png}} -l {{program lanauge}}`
```

Comment:
There is no list of languages but it could be `javascript` or `js`.
```suggestion
`silicon  {{path/to/source_file}} --output {{path/to/output_image.png}} --language {{language_or_file_extension}}`
```